common/src/mailbox_api.rs

```diff
@@ -270,6 +297,14 @@ pub struct StashMeasurementResp {
 // No command-specific input args
 // No command-specific output args
 
+// INCREMENT_PCR_RESET_COUNTER
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct IncrementPcrResetCounterReq {
```

Comment:
This is currently missing from the spec, but all Caliptra mailbox responses should start with `chksum` and `fips_status` fields. Could you update this PR with:

1. Add response structs for all commands
2. Update spec
3. Add these fields